emspy/query/fltquery.py

```diff
@@ -557,7 +557,7 @@ def __to_dataframe(self, json_output):
                 elif ctype == 'boolean':
                     df.iloc[:, i] = df.iloc[:, i].astype(bool)
                 elif ctype == 'dateTime':
-                    df.iloc[:, i] = pd.to_datetime(df.iloc[:, i]).dt.tz_localize('UTC')
+                    df.iloc[:, i] = pd.to_datetime(df.iloc[:, i]).dt.tz_convert('UTC')
```

Comment:
Hey Matt, 

I tested this a bit more today, and I found that older versions of pandas prefer ```tz_localize()```, whereas newer versions prefer ```tz_convert()```. It looks like the cutoff is about pandas version == 0.21.0. Just based on my limited testing. So before 0.21.0, ```tz_convert('UTC')``` fails, and after 0.21.0 (as you found) ```tz_localize('UTC')``` fails. 

I'm wondering if we even need to do any conversion at all. It looks like ```pd.to_datatime()``` accepts a ```utc=True``` argument. That works for all of the Pandas versions I've tested (0.18.0, 0.21.0, and 1.1.4 [most recent]). 

So I think my preference (if it works for you) would be to change line 560 to: 

```python
df.iloc[:, i] = pd.to_datetime(df.iloc[:, i], utc=True)
```

Can you see if that works in your scripts, and if so, push your changes to the same mattsq:master branch? That should update this pull request and we can get this merged. 

Thanks, 

Daniel

Reply:
That makes sense! I'll rebuild it and see how we go.

Reply:
@danverkamp done!